accessible/clipboard.service.js

```diff
@@ -0,0 +1,98 @@
+var blocklyApp = blocklyApp || {};
```

Comment:
also, why do we keep redefining blocklyApp? In the case where blocklyApp isn't already present, shouldn't there be an error instead?